### PR TITLE
auto 4.2 caseOCP-23681

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.1/browser_opts.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+:scroll_strategy: 1
```
Comment (yapei) [inline lib/rules/web/admin_console/4.1/browser_opts.yml]: Looks like this file is missed when 4.1 rules were copied, bring it back

**Comment by yapei** [comment]
@yanpzhan  @SSshahan  Can you please help review? Thanks

File: lib/rules/web/admin_console/4.2/actions.xyaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+click_one_dropdown_action:
+  action: click_resource_action_button
+  action: choose_item_from_list
+click_resource_action_button:
+  elements:
+  - selector:
+      xpath: //button[contains(.,'Actions')]
```
Comment (yapei) [inline lib/rules/web/admin_console/4.2/actions.xyaml]: rules is changed from `/button//span[contains(text(),‘Actions’)]` to `//button[contains(.,'Actions')]`, when using `/button//span[contains(text(),‘Actions’)]` overlap is found and element is not clickable

File: lib/rules/web/admin_console/4.2/storage.xyaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  - selector:
+      id: storageclass-dropdown
+    type: button
+    op: click
+    timeout: 20
+  - selector:
+      id: <storage_class_name>-link
+    type: a
+    op: click
+set_pvc_name:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.2/storage.xyaml]: line 24 - line 70 is newly added

**Comment by akostadinov** [comment]
LGTM from automation and reliability point of view. Steps and checkpoints may need additional review by feature team members.

-- saved reply

File: lib/rules/web/admin_console/4.2/common_ui_elements.xyaml [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+  elements:
+  - selector:
+      xpath: //button[contains(text(),'Create')]
+    op: click
+click_cancel_button:
+  elements:
+  - selector:
+      xpath: //button[contains(text(),'Cancel')]
+    op: click
+click_button:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.2/common_ui_elements.xyaml]: `click_button_on_page` is changed to `click_button` with parameter `<text>` changed to `<button_text>` @SSshahan

Comment (yapei) [reply lib/rules/web/admin_console/4.2/common_ui_elements.xyaml]: https://github.com/openshift/verification-tests/blob/master/lib/rules/web/admin_console/4.1/common_ui_elements.xyaml#L27

Comment (SSshahan) [reply lib/rules/web/admin_console/4.2/common_ui_elements.xyaml]: Do we need update this rule in 4.1?

**Comment by SSshahan** [comment]
It looks good to me. Thanks.

**Pull request merged**